host_cli/data_core_wrapper.cpp

```diff
@@ -66,4 +67,26 @@ absl::Status DataCoreWrapper::WriteNewGfxrFile(const std::string& new_gfxr_file_
     return absl::OkStatus();
 }
 
+absl::Status DataCoreWrapper::RemoveGfxrBlocks(std::vector<int> block_ids)
+{
+    assert(m_data_core != nullptr);
+    if (!IsGfxrLoaded())
+    {
+        return absl::FailedPreconditionError("Must load original GFXR first");
+    }
+
+    std::shared_ptr<gfxrecon::decode::DiveBlockData>
+    dive_block_data = m_data_core->GetMutableCaptureData().GetMutableGfxrData();
+
+    for (const auto& id : block_ids)
+    {
+        bool res = dive_block_data->AddModification(id, 0, nullptr);
```

Comment:
nit: Consider function argument comments to make this more readable. I had to go to the definition of AddModification to know that `0` and `nullptr` were for.
```suggestion
        bool res = dive_block_data->AddModification(id, /*secondary_id=*/0, /*blob_ptr=*/nullptr);
```